xbmc/dialogs/GUIDialogSmartPlaylistEditor.cpp

```diff
@@ -345,50 +400,50 @@ void CGUIDialogSmartPlaylistEditor::UpdateButtons()
     SET_CONTROL_LABEL2(CONTROL_NAME, m_playlist.m_playlistName);
   
   UpdateRuleControlButtons();
+  
 
+  if (m_playlist.m_ruleCombination.GetType() == CSmartPlaylistRuleCombination::CombinationOr)
+    SET_CONTROL_LABEL2(CONTROL_MATCH, g_localizeStrings.Get(21426));
```

Comment:
Could you add comments to these two `SET_CONTROL_LABEL2` lines to know what the new labels are?